MediaBrowser.MediaEncoding/MediaBrowser.MediaEncoding.csproj

```diff
@@ -17,4 +17,15 @@
     <ProjectReference Include="..\OpenSubtitlesHandler\OpenSubtitlesHandler.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.5.1" />
+    <PackageReference Include="UTF.Unknown" Version="1.0.0-beta1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Text.Encoding.CodePages, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
```

Comment:
Why is this needed?

Reply:
Drone complained before. It's fine after the rebase... dunno. Maybe I was missing an import.